test/instrumentations/mongodb/mongodb.test.ts

```diff
@@ -31,27 +31,56 @@ const expectedIndexStatement = expect.stringMatching(
 );
 
 describe.each(versionsToTest('mongodb', 'mongodb'))(
-  "Instrumentation tests for the 'mongodb' package",
-  function (versionToTest) {
-    let testApp: ChildProcessWithoutNullStreams;
+  'Instrumentation tests for the mongodb package',
+  (versionToTest) => {
```

Comment:
there's a reason that i switched away from all the arrow functions - in both jest and mocha, arrow functions break things. unless there's some important reason for arrow functions here that i'm not familiar with, please return these

Reply:
Done in 367dcee